packages/expo-device/src/ModelNames.ts

```diff
@@ -80,17 +80,22 @@ export const iosModelNames = {
   'iPhone12,1': 'iPhone 11',
   'iPhone12,3': 'iPhone 11 Pro',
   'iPhone12,5': 'iPhone 11 Pro Max',
+  'iPhone12,8': 'iPhone SE', // 2nd Generation iPhone SE
```

Comment:
```suggestion
  'iPhone12,8': 'iPhone SE (2nd generation)',
```
any reason not to do this?